### PR TITLE
Fix storage queues sample issues

### PR DESCRIPTION
**Pull request by LuHang2021**
Fix issue https://github.com/Azure/azure-sdk-for-java/issues/21048

@jongio for notification.

### CODE REVIEW

File: sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/QueueAsyncJavaDocCodeSamples.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ public void enqueueMessageAsync() {
      */
     public void enqueueMessageBinaryDataAsync() {
         // BEGIN: com.azure.storage.queue.queueAsyncClient.sendMessage#BinaryData
-        client.sendMessage(BinaryData.fromString("Hello, Azure")).subscribe(
+        client.sendMessage(BinaryData.fromString("Hello, Azure").toString()).subscribe(
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/QueueAsyncJavaDocCodeSamples.java:148]: The issue may be slightly incorrect as there are two `sendMessage` APIs, one accepts `String` and the other accepts `BinaryData`.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java#L419

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueClient.java#L442

Comment (LuHang2021) [reply sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/QueueAsyncJavaDocCodeSamples.java]: @alzimmermsft I have updated

**Comment by alzimmermsft** [comment]
@gapra-msft FYI

**Review by alzimmermsft — APPROVED** [review]

**Review by gapra-msft — APPROVED** [review]

**Comment by v-xuto** [comment]
@alzimmermsft - Could you help us merge this PR?

**Pull request merged**